README.md

```diff
@@ -33,8 +33,6 @@ terraform apply -var-file="terraform.tfvars"
 
```

Comment:
Can you add the name of the project at the top of this file too. Right now it only says "Project Name"

Reply:
Yes, good point @edaena, done! Update description as well.